src/openmct-yamcs.js

```diff
@@ -89,6 +90,11 @@ export default function install(configuration) {
             configuration.yamcsHistoricalEndpoint,
             configuration.yamcsInstance));
 
+        const missionStatusTelemetry = new MissionStatusTelemetry(openmct, {
+            url: configuration.yamcsHistoricalEndpoint,
+            instance: configuration.yamcsInstance,
+            styleConfig: configuration.statusStyles
```

Comment:
These `statusStyles` should be different from the operator status styles. Likely need to create a new config for this.

Reply:
good catch, this remains to be implemented

Reply:
Can you create an issue for this and link it here?

Reply:
Actually I'm going to move this comment to Open MCT. That's probably where the config should be anyway.